pages/1.13/installing/production/advanced-configuration/configuration-reference/index.md

```diff
@@ -587,6 +589,13 @@ The SSH username, for example `centos`.
 
 **Note:** The ssh_user is not required for the production installer.
```

Comment:
Rewrite Note using the below format (copy-paste the below line):
<p class="message--note"><strong>NOTE: </strong>The `ssh_user` is not required for the production installer.</p>